docs/host_for_multiuser/setup.md

```diff
@@ -110,20 +119,27 @@ docker compose -f docker-compose.dev.multiuser.yml up studio-dev-be -d
 
 ###### 3. Insert initial data
 ```bash
-docker exec -it {DB_DOCKER_CONTAINER_NAME} mysql -u {DB_USER_NAME} -p {YOUR_DATABASE_NAME}
+docker exec -it {DB_CONTAINER_NAME} mysql -u {DB_USER_NAME} -p {YOUR_DATABASE_NAME}
 ```
 ```sql
 INSERT INTO organization (name) VALUES ('{YOUR_ORGANIZATION_NAME}');
 INSERT INTO roles (id, role) VALUES (1, 'admin'), (20, 'operator');
-INSERT INTO users (uid, organization_id, name, email, active) VALUES ('{FIREBASE_UID}', 1, '{YOUR_NAME}', '{YOUR_EMAIL}', true);
+INSERT INTO users (uid, organization_id, name, email, active) VALUES ('{FIREBASE_USER_UID}', 1, '{YOUR_NAME}', '{YOUR_EMAIL}', true);
 INSERT INTO user_roles (user_id, role_id) VALUES (1, 1);
 ```
-- USER_UID is the user uid you created in the previous step ([Create admin user for the project](#create-admin-user-for-the-project)).
-- Only 2 roles, "admin" and "operator" are supported for now.
-  - "admin"
-    - can manage other users
-  - "operator"
-    - general user
+  - Note of Variables
```

Comment:
This is much clearer. Thank you!

Note **on** Variables

Reply:
Corrected wording.